bedrock/mozorg/views.py

```diff
@@ -162,6 +162,75 @@ def contribute_signup(request):
     return view_class.as_view()(request)
 
 
+class ContributeTasks(l10n_utils.LangFilesMixin, TemplateView):
+    variation_re = re.compile('^[1-4]$')
+
+    def get_context_data(self, **kwargs):
+        cxt = super(ContributeTasks, self).get_context_data(**kwargs)
+        variation = self.request.GET.get('variation', '4')
+        match = self.variation_re.match(variation)
+        if match:
+            cxt['variation'] = variation
+        return cxt
+
+    template_name = 'mozorg/contribute/contribute-tasks.html'
```

Comment:
NIT: Odd to have a class variable after a method definition. Typically all class variable definitions would be at the top of the class.